src/main/java/study/racingcar/domain/Name.java

```diff
@@ -27,7 +25,7 @@ private void validateName(String name) {
 	}
 
 	private void checkNameBlank(String name) {
-		if (StringUtils.isBlank(name)) {
+		if(" ".equals(name)){
```

Comment:
```suggestion
		if (" ".equals(name)) {
```
1. 코딩 컨벤션 및 포맷팅 확인 부탁드릴게요.
2. 빈 문자열인 `""`에 대해서도 검증이 되었으면 좋겠습니다.
`" "`같이 공백이 포함된 경우는 검증하기 전에 `.trim()`을 호출하면 쉽게 처리 가능할 것 같아요.
지금은 `InputView.stringIntoNames()`에서만 `.trim()`이 호출되고 있지만요.